js/scriptContacto.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+    
+    (function titleSize(){
+        let windowWidthO = window.outerWidth;
+        let windowWidthI = window.innerWidth;
+        let titulo = document.getElementById("titulo");
+        let fontSize = window.getComputedStyle(titulo).getPropertyValue('font-size');
+        
+        console.log(windowWidthO);
+        console.log(windowWidthI);
+        
+        if(windowWidthO < 1000){

```

Comment:
Codacy has a fix for the issue: [Replace `········if(windowWidthO·<·1000){␍` with `····if·(windowWidthO·<·1000)·{`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456241)
```suggestion
    if (windowWidthO < 1000) {
```